server/routes/index.js

```diff
@@ -17,6 +19,13 @@ router.get('/:code', async (req, res) => {
             const longUrl = url["urls"][0]["longUrl"];
             if (longUrl) {
                 console.log(longUrl);
+
+                // Use the hosts file to check if any unsafe URL is being shortened
+                const safe = await hostCheck.checkSafeURL(longUrl);
+                if(!safe) {
+                  return res.status(404).json({"error": "This URL was blocked"});
+                }
+
```

Comment:
@riskycase 
Instead of using this section of code again and again in routes/index.js routes/user.js routes/url.js
You can create a utility function by importing hostCheck
`const hostCheck = require('../hosts/hostCheck');`
 into that file and writing the same logic and export it as a module that can be reused.
This would make the code cleaner and more modular as well.

Reply:
Reduced each instance to a single line by sending the status code via the hosts file